### ISSUE TITLE
app local profiles should be placed before global local profiles

### ISSUE THREAD

**Issue by VladimirSchowalter20** [netblue30/firejail#1434]
Currently local profiles are sourced in this precedence:
```
# Persistent global definitions go here
include /etc/firejail/globals.local

# This file is overwritten during software install.
# Persistent customizations should go in a .local file.
include /etc/firejail/ark.local

```

It means that when user enables some option in `global.local `profile it can't be overridden in `ark.local` profile. Therefore any global option which breaks one app (example:` apparmor`) can be overridden only by editing `ark.profile` directly which will be overwritten during next update or by creating new profile in `~/.config/firejail/ark.profile` which have to be then maintained solely by user.

Basing on the above I propose changing precedence to:
```
# This file is overwritten during software install.
# Persistent customizations should go in a .local file.
include /etc/firejail/ark.local

# Persistent global definitions go here
include /etc/firejail/globals.local

```
This way user can override any `global.local` option in `ark.local` file easily. As both `global.local` and `ark.local` are used only for personal local administration this shouldn't create any security issues.

**Comment by netblue30** [comment]
Sure, it makes sense. Let's wait for @Fred-Barclay and @SpotComms to finish with the current round of cleanups, and I'll write some small script to go through all the files and switch the lines.

**Comment by SpotComms** [comment]
This is actually already fixed in #1427 since sometime after you opened this.
